src/operations/search_indexes/create.ts

```diff
@@ -8,12 +8,15 @@ import { AbstractOperation } from '../operation';
 /**
  * @public
  */
-export interface SearchIndexDescription {
+export interface SearchIndexDescription extends Document {
```

Comment:
We can remove this, or do it in a separate PR.  It was an oversight in the initial implementation not to allow arbitrary keys on the `SearchIndexDescription` for future compatibility, so I added it now.